gobblin-docs/user-guide/Gobblin-CLI.md

```diff
@@ -35,23 +35,27 @@ options:
                 keystore                    Examine JCE Keystore files
                 watermarks                  Inspect streaming watermarks
                 job-store-schema-manager    Database job history store schema manager
-
-    --conf-dir <gobblin-conf-dir-path> Gobblon config path. default is '$GOBBLIN_HOME/conf/<exe-mode-name>'.
-    --log4j-conf <path-of-log4j-file>  default is '<gobblin-conf-dir-path>/<execution-mode>/log4j.properties'.
-    --jvmopts <jvm or gc options>      String containing JVM flags to include, in addition to "-Xmx1g -Xms512m".
-    --jars <csv list of extra jars>    Column-separated list of extra jars to put on the CLASSPATH.
-    --enable-gc-logs                   enables gc logs & dumps.
-    --show-classpath                   prints gobblin runtime classpath.
-    --help                             Display this help.
-    --verbose                          Display full command used to start the process.
-                                       Gobblin Version: 0.15.0
+                gobblin-classpath           shows the constructed gobblin classpath
+
+    --conf-dir <gobblin-conf-dir-path>    Gobblon config path. default is '$GOBBLIN_HOME/conf/cli'.
+    --log4j-conf <path-of-log4j-file>     default is '<gobblin-conf-dir-path>/cli/log4j.properties'.
+    --work-dir <gobblin-work-dir>         Gobblin work dir. default is /Users/jsenjaliya/tools/gobblin-dist/gobblin-cluster-data.
+    --job-conf-dir <path-of-log4j-file>   Job configuration dir to pick up the jobs to run. default is /Users/jsenjaliya/tools/gobblin-dist/gobblin-jobs.
+    --jvmopts <jvm or gc options>         JVM or GC parameters for the java process to append to the default params: "-Xmx1g -Xms512m".
+    --jars <csv list of extra jars>       Column-separated list of extra jars to put on the CLASSPATH.
+    --enable-gc-logs                      enables gc logs & dumps.
+    --show-classpath                      prints gobblin runtime classpath.
+    --help                                Display this help.
+    --verbose                             Display full command used to start the process.
 ```
 
 
 Argument details:
-* `--conf-dir`: specifies the path to directory containing gobblin system configuration files, like `application.conf` or `reference.conf`, `log4j.properties` and `quartz.properties`.
+* `--conf-dir`: specifies the path to directory containing gobblin system configuration files, like `application.conf` or `reference.conf`, `log4j.properties` and `quartz.properties`, as this is provided as classpath for config file look up.
```

Comment:
I find this addition confusing, what do you mean by `provided as classpath?`

Reply:
meant `--conf-dir` value is supplied as classpath for any config file to be looked up.